src/analysisd/decoders/security_configuration_assessment.c

```diff
@@ -978,11 +977,21 @@ static void HandleScanInfo(Eventinfo *lf,int *socket,cJSON *event) {
             os_calloc(OS_MAXSTR, sizeof(char), old_hash);
             if(FindPolicySHA256(lf, policy_id->valuestring, socket, old_hash) == 0){
                 if(strcmp(hash_file->valuestring, old_hash)){
-                    DeletePolicyCheck(lf, policy_id->valuestring, socket);
-                    DeletePolicy(lf, policy_id->valuestring, socket);
-                    PushDumpRequest(lf->agent_id, policy_id->valuestring, 1);
+                    int delete_status = DeletePolicy(lf, policy_id->valuestring, socket);
+                    switch (delete_status) {
+                        case 0:
+                            /* Delete checks */
+                            DeletePolicyCheck(lf, policy_id->valuestring, socket);
+                            mdebug2("Policy '%s' deleted. Requesting SCA new scan", policy_id->valuestring);
```

Comment:
`minfo("DB content for policy '%s' outdated. Latest scan requested to the SCA module.", policy_id->valuestring);`